ansible/roles/tools/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+- name: Download ngrok
+  unarchive:
+    src: https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-arm.zip
+    dest: /home/pi/screenly/bin/
+    remote_src: yes
+
+- name: Installs nginx config
+  copy:
+    src: nginx.conf
+    dest:  /etc/nginx/sites-enabled/screenly_assets.conf
+    mode: 644
+    owner: root
+    group: root
+
+- name: Add custom-path to /etc/profile.d.
```

Comment:
Why is this required?

Reply:
In order to use `sh.ngrok`
What do you think about this?

Reply:
I'd recommend that you refactor this to either:
 * Use the `ngrok-client` apt package (if available)
 * Copy it to `/usr/local/bin/`